proposed/http-middleware/middleware-meta.md

```diff
@@ -222,8 +222,8 @@ and a delegate and must return a response. The middleware may:
 - Evolve the request before passing it to the delegate to execute the next
   available middleware.
 - Evolve the response received from the delegate before returning it.
-- Create and return a response without passing it to the delegate, thereby
-  preventing any further middleware from processing.
+- Create and return a response without further process delegation, thereby
+  preventing any inner middleware from processing.
```

Comment:
The word `inner` is currently in use in the document. See: https://github.com/php-fig/fig-standards/blame/master/proposed/http-middleware/middleware-meta.md#L129

The relationship `inner` `outer` is established by the delegate on a run time. If the request processing is delegated by the `Hello-World middleware` to the `FOO middleware` then the `FOO middleware` is the `inner` middleware. The source code visual organization is not relevant.

It could be good tough to clarify the language related to the middlewares even further by defining this relationships in the document itself.